src/controllers/files/shared_with_who/_handler.py

```diff
@@ -0,0 +1,40 @@
+import json
+from flask import request
+from src.config.soap_client import soap_client
+from src.lib.helpers import is_valid_uuid
+
+
+def shared_with_who_handler(token):
+    try:
+        data = json.loads(request.data)
```

Comment:
As this is a GET endpoint, it does not receive any JSON data. You'll receive the fileUUID as a controller argument, similar to the [download endpoint](https://github.com/hawks-atlanta/proxy-python/pull/60) you implemented.

Could you update the spec as well? The endpoint should be something like `/file/{fileUUID}/shared-with-who`.